core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -43,6 +43,7 @@
 import org.apache.calcite.util.TimeWithTimeZoneString;
 import org.apache.calcite.util.TimestampWithTimeZoneString;
 
+import com.google.common.base.Optional;
```

Comment:
Do we need to use Guava here? Java 8 optional should work?

Reply:
Thanks @risdenk. We can use Java 8. I have made the change.